src/errors/custom.error.ts

```diff
@@ -24,3 +18,19 @@ export abstract class CustomError extends Error {
         }
     }
 }
+
+
```

Comment:
All custom error implementations exist side by side their usage. The idea being that particular Error type is used only from a specific helper. I assume you plan to use APIError in multiple places, if so just create another file and add this in.